OpenRA.Mods.Common/Traits/LevelupWhenCreated.cs

```diff
@@ -0,0 +1,49 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Actors possessing this trait should define the GainsExperience trait. When the prerequisites are fulfilled, ",
+		"this trait grants a level-up to newly spawned actors. If additionally the actor's owning player defines the CanUpgradeIcons ",
```

Comment:
I believe the part about `CanUpgradeIcons` is outdated? That is now `ProductionIconOverlay`, I think.